requirements.txt

```diff
@@ -1 +1,2 @@
+iugu
```

Comment:
We don't need iugu here or do we?

Reply:
You are right, this dependency is in "odoo-brasil", but not in the l10n_br_automated_payment manifest. I did a PR putting this.